specs/createAccountNegative.spec.js

```diff
@@ -0,0 +1,27 @@
+let randomstring = require('randomstring');
+let homePage = require('../pageObject/homePage');
+let loginPage = require('../pageObject/loginPage');
+let createAccountPage = require('../pageObject/createAccountPage');
+
+let mail = randomstring.generate(7);
+
+
+describe('Create Account', () => {
+    it('Negative', async () => {
+        await homePage.get();
+        await homePage.clickLogin();
+        await loginPage.createAccount(`apptest709@gmail.com`);
+        await browser.sleep(2000);
```

Comment:
Please avoid browser.sleep, if need please use browser.wait for some element or condition